Source/WebKit/NetworkProcess/NetworkProcess.h

```diff
@@ -674,6 +674,10 @@ class NetworkProcess final : public AuxiliaryProcess, private DownloadManager::C
         CompletionHandler<void()> completionHandler;
     };
     HashMap<TaskIdentifier, DeleteWebsiteDataTask> m_deleteWebsiteDataTasks;
+
+#if ENABLE(DNS_SERVER_FOR_TESTING_IN_NETWORKING_PROCESS)
+    OSObjectPtr<nw_resolver_config_t> m_resolverConfig;
```

Comment:
Is this change about fixing a lifetime issue? If so do you want to separate that out and get that land first?